src/Forte.Optimizely.ContentUsage/Api/Features/ContentType/ContentTypeUsagesDB.cs

```diff
@@ -0,0 +1,113 @@
+using System;
+using System.Collections.Generic;
+using System.Data.Common;
+using EPiServer.Construction;
+using EPiServer.Core;
+using EPiServer.Data;
+using EPiServer.DataAbstraction;
+using EPiServer.DataAccess.Internal;
+using EPiServer.Framework.Blobs;
+using EPiServer.ServiceLocation;
+using EPiServer.Web;
+
+namespace Forte.Optimizely.ContentUsage.Api.Features.ContentType;
+
+[ServiceConfiguration(Lifecycle = ServiceInstanceScope.Singleton)]
```

Comment:
can we register this in regular DI way by `services.AddSingleton` in `AddContentUsage` method?